### DISCUSSION TITLE
Some functions from orcaslicer

### DISCUSSION THREAD

**Discussion by Serj121** [supermerill/SuperSlicer#4091] [General]
Tell me, do you have the opportunity to add some functions from orcaslicer?
for example
![image](https://github.com/supermerill/SuperSlicer/assets/124658770/30853e37-7309-46dc-b2f0-f34c6ad205ac)
![image](https://github.com/supermerill/SuperSlicer/assets/124658770/10dbac2e-4e8e-49e5-81e8-798cc5381f28)

**Comment by supermerill** [comment]
I will, but for now I'm finishing merging prusa 2.7
